paddle/fluid/framework/ir/mkldnn/operator_scale_onednn_fuse_pass.cc

```diff
@@ -86,17 +87,8 @@ void FuseOperatorScaleOneDNNPass::FuseScale(Graph *graph,
       scale = *(scale_tensor->data<float>());
     }
 
-    if (op_type == "matmul") {
-      operator_op->Op()->SetType("fused_matmul");
-      operator_op->Op()->SetAttr("trans_x",
-                                 operator_op->Op()->GetAttr("transpose_X"));
-      operator_op->Op()->SetAttr("trans_y",
-                                 operator_op->Op()->GetAttr("transpose_Y"));
-      operator_op->Op()->SetAttr("matmul_alpha",
-                                 operator_op->Op()->GetAttr("alpha"));
-    }
-    if (op_type == "matmul_v2") {
-      operator_op->Op()->SetType("fused_matmul");
+    if (op_type == "matmul" || op_type == "matmul_v2") {
+      ConvertToFusedOp(operator_op->Op());
```

Comment:
This check will be done in `ConvertToFusedOp` function.
```suggestion
    ConvertToFusedOp(operator_op->Op());
```

Reply:
I left these checks in fuse passes which affect ops, that don't have fused op implemented yet. For example in _operator_scale_onednn_fuse_pass_ only 2 of 7 ops (matmuls) can be converted to fused op. There is no reason to call this function for `fc` or `elementwise` ops until `fused_fc` and `fused_elementwise` are implemented.

But I agree that later these extra checks should be deleted in all cases you've mentioned in code review.